src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -376,6 +378,7 @@ externalizeFreshVars pat = Recursive.fold renameFreshLocal pat
 main :: IO ()
 main = do
     options <- mainGlobal parseKoreExecOptions parserInfoModifiers
+    threadDelay 10000000
```

Comment:
Why do we need this long delay on startup?

Reply:
Ooops, this was some debugging instrumentation that I failed to remove, will send a PR.